locales/ja.json

```
@@ -38,20 +38,20 @@
         "hero": "ALCOM は VRChat Creator Companion (VCC) のオープンソースな非公式代替ツールです。<br>Windows、macOS、Linux のいずれでも利用できます。",
         "promotions": {
             "fastInstallation": {
-                "title": "パッケージのインストールに長時間待たされることはありません",
-                "content": "<p>各パッケージをインストールするのに何十秒も待つ必要はもうありません。複数のパッケージを一度にインストールすることができ、パッケージのアップグレードもほとんどの場合数秒で済みます</p>"
+                "title": "パッケージを素早くインストール",
+                "content": "<p>使いたいパッケージをインストールするために何十秒も待たされることはもうありません！複数のパッケージを一度にインストール・更新することができ、ほとんどの場合は数秒待つだけで済みます。</p>"
             },
             "accessToChangelog": {
-                "title": "変更履歴への容易なアクセス",
-                "content": "<p>更新内容を確認するためにウェブサイトを探す必要はもうありません。パッケージが指定している場合、 ALCOM はパッケージのインストール時に CHANGELOG へのリンクを表示します。</p>"
+                "title": "更新内容を簡単に閲覧",
```

```suggestion
                "title": "簡単な更新内容の閲覧",
```

名詞句のほうが自然な気がしてこれとか思いついたけど要議論

---

インストール速度の方、"パッケージを素早くインストール"ってしたから、名詞句じゃなくても統一感的には問題ないかもしれない

---

確かにインストールは名詞句じゃないな。

いじらないことにします